js/components/volumeControls.js

```diff
@@ -0,0 +1,195 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var ClassNames = require('classnames');
+var Slider = require('./slider');
+var Utils = require('./utils');
+var CONSTANTS = require('../constants/constants');
+
+var VolumeControls = React.createClass({
+
+  handleVolumeClick: function(event) {
+    event.preventDefault();
+    var newVolume = parseFloat(event.target.dataset.volume);
```

Comment:
Do we need a null check for event.target.dataset?

Reply:
I copied this straight from the old code, I think it's a good idea to add the check though